### PR TITLE
Create, populate and query animals table 

### PR DESCRIPTION
**Pull request by JonahKayizzi**
This pull request consists of the following;

- Created the `vet_clinic` database
- Created `animals` table
- Inserted data into the `animals` table
- Queried the following quests from the `animals` table

1. Find all animals whose name ends in "mon"
image

2. List the name of all animals born between 2016 and 2019.
image

3. List the name of all animals that are neutered and have less than 3 escape attempts.
image

4. List the date of birth of all animals named either "Agumon" or "Pikachu".
image

5. List name and escape attempts of animals that weigh more than 10.5kg
image

6. Find all animals that are neutered.
image

7. Find all animals not named Gabumon.
image

8. Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(50),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BIT,
```
Comment (codecaiine) [inline schema.sql]: - [ ] According to project requirements the column `neutered ` is `boolean` not `BIT`. a bit type only stores the numbers 0 and 1 (or null). note that the boolean only stores true and false (or null). A number (0, 1) is not a boolean. A boolean value can be used anywhere a boolean expression is expected. Please fix it :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @JonahKayizzi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good use gitflow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (JonahKayizzi) [reply schema.sql]: Thank you for this correction. The change has been updated

**Review by codecaiine — APPROVED** [review]
Hi @JonahKayizzi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of gitflow :heavy_check_mark: 
- Good database setup :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**